packages/www/docs/guides/upload-a-video/index.mdx

```diff
@@ -0,0 +1,775 @@
+---
+title: Upload a Video
+description: Livepeer's Upload a Video Guide
+metaTitle: Upload a Video Guide - Livepeer.com
+metaDescription: Livepeer's Upload a Video Guide
+---
+
+# Upload a Video
+
+This tutorial covers how to upload the video to Livepeer to allow viewers to
+playback stored video assets in your application.
+
+## Caveats
+
+- Have to manually check the status of the file using `List Task` method to see
+  if it has finished uploading.
+- File size is limited to 1GB. Any files greater than 1GB will error with
+  NETWORK CHANGED.
+- Only MP4 files encoded with H.264 and AAC are supported.
+
+## Installation
+
+- Install [Node.js](https://nodejs.org/en/)
+- Install package manager with
+  [NPM](https://docs.npmjs.com/cli/v8/configuring-npm/install) or
+  [Yarn](https://classic.yarnpkg.com/en/docs/install#mac-stable)
+- Install [Axios](https://axios-http.com/)
+  - Using npm
+  ```bash
+  npm axios
+  ```
+  - Using yarn
+  ```bash
+  yarn add axios
+  ```
+- Get a
+  [Livepeer Video Service API key](https://livepeer.com/docs/guides/start-live-streaming/api-key)
```

Comment:
I think all these steps are unnecessary. This is a technical documentation for developers getting started with the VOD API. If they are using JavaScript, they already have `node` and a package manager and know how to use it. I'd just make a remark here that we will be using Node and `axios` so they can install it on their projects (IMO don't need to teach them how to do it either).

This guide is also inside the "Livepeer Video Services" website, so we should not keep repeating that. It's very verbose/cumbersome to read so it's better to avoid it.

```suggestion
## Dependencies

In this tutorial, we will be using [Node.js](https://nodejs.org/en/) and [Axios](https://axios-http.com/) so make sure to set that up in your project. You will also need to [Get an API key](https://livepeer.com/docs/guides/start-live-streaming/api-key) to call the VOD API.
```